### PR TITLE
Event creation date.

### PR DESCRIPTION
**Pull request by miskosz**
Introduced `creation_date` so that we don't need to arbitrarily add one day. Also changes sorting countries in the dropdown by displayed label.

### CODE REVIEW

**Comment by miskosz** [comment]
@rejdi I touched `Rest` interface, would be great if you could have a look.

File: covid_graphs/covid_graphs/scripts/rest.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def _create_predictions(
                         "max_value": trace.max_value,
                         "date_name": prediction.prediction_event.name,
                         "date": prediction.prediction_event.date,
+                        "creation_date": prediction.prediction_event.creation_date,
                     }
```
Comment (rejdi) [inline covid_graphs/covid_graphs/scripts/rest.py]: Addition to the interface is no problem. I'm good to go.

File: covid_graphs/covid_graphs/predictions.py [hunk 2]
```diff
@@ -9,7 +9,9 @@
 @dataclass(frozen=True, eq=True)
 class PredictionEvent:
     name: str
+    # Data until this date (inclusive) was used for the prediction.
     date: datetime.date
+    creation_date: datetime.date
```
Comment (rejdi) [inline covid_graphs/covid_graphs/predictions.py]: The comment saves the day for not very usefull name. I'd consider something like `date` -> `last_data_date` and `creation_date` -> `prediction_date` or simply `date` as it's already part of PredictionEvent class. Until now, I was thinking that the old `date` was date of prediction and not last data date used for prediction :smile:

**Review by lukipuki — APPROVED** [review]

File: covid_graphs/covid_graphs/predictions.py (hunk 2 above)
Comment (lukipuki) [reply covid_graphs/covid_graphs/predictions.py]: Agree here

Comment (miskosz) [reply covid_graphs/covid_graphs/predictions.py]: `last_data_date` is a great naming suggestion. I'll make the change (also creation_date -> prediction_date).

**Pull request merged**